code/script.js

```diff
@@ -30,18 +34,114 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Starts here with a start button leading to the chat and inital greeting (with timer)
+
+startBtn.onclick = () => {
+  startPage.style.display = "none"
+  setTimeout(greeting, 1200)
+}
+
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hi there, who is this?`, 'bot') // bot greeting here
+}
+
+const handleNameInput = (event) => {
+  event.preventDefault();
+  const name = nameInput.value;
+  showMessage(`It's me, ${name}. Please help!`, 'user');
+  nameInput.value = "";
+  setTimeout(() => showDecade(name), 1000);  // users first answer, when submitting answer peage refresh is prevented by (event)
 }
 
-// Set up your eventlisteners here
+const showDecade = (name) => { // follow upp question by bot, showing first options buttons
+  showMessage(`Welcome ${name}. Of course! First, please tell me what decade you feel like diving into?`, 'bot');
+  setTimeout(() => selectMovie(year), 1000); 
+
+  inputWrapper.innerHTML = `
+    <button id='1980Btn'>1980s</button>
+    <button id='1990Btn'>1990s</button>
+    <button id='2000Btn'>2000s</button>
+  `
+
+  document.getElementById('1980Btn').addEventListener("click", () => selectMovie('1980s')); // when one of these selections are selected, it leads the chat to the next question which is the selectMovie function 
+  document.getElementById('1990Btn').addEventListener("click", () => selectMovie('1990s'));
+  document.getElementById('2000Btn').addEventListener("click", () => selectMovie('2000s'));
+}
+  
+const selectMovie = (year) => { 
+  inputWrapper.innerHTML ="" // this clears input wrapper from content (the buttons in the previous question)
+  showMessage(`Really feel like watching something from the ${year} `,  'user');
+  showMessage(`${year} you say! Cool, what genre do you feel like?`, 'bot');
```

Comment:
To follow a similiar structure as the previous questions, consider pulling out this code and creating a new `selectGenre` function that can be called at the end of `selectMovie` with a delay using `setTimout`